utils/src/tree/AzExtTreeItem.ts

```diff
@@ -118,7 +120,11 @@ export abstract class AzExtTreeItem implements types.AzExtTreeItem {
     }
 
     public get tooltip(): string | undefined {
-        return this._tooltip;
+        if(process.env.DEBUGTELEMETRY === 'v' && settingUtils.getWorkspaceSetting(showContextValueSetting) === true) {
```

Comment:
I believe it would get reflected on any item that has not yet been rendered but have no effect on items that have already been constructed unless reloaded or reconstructed